domain/learningandworkprogress/src/test/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/service/InductionScheduleDateCalculationServiceTest.kt

```diff
@@ -7,14 +7,24 @@ import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.CsvSource
 import uk.gov.justice.digital.hmpps.domain.learningandworkprogress.induction.InductionScheduleStatus
 import uk.gov.justice.digital.hmpps.domain.learningandworkprogress.induction.aValidInductionSchedule
+import uk.gov.justice.digital.hmpps.domain.learningandworkprogress.induction.dto.CreateInductionScheduleDto
 import java.time.LocalDate
 
 class InductionScheduleDateCalculationServiceTest {
   companion object {
     private val TODAY = LocalDate.now()
   }
 
-  private val dateCalculationService = InductionScheduleDateCalculationService()
+  // New up an anonymous instance of the abstract class to test the methods implemented in the abstract class
```

Comment:
I want to keep the main unit test of the methods implemented in the abstract class within the domain module, because thats really where it lives 👍 
The testing of the implemented abstract methods in the PES and PEF implementations via their own unit test classes